app/testing/page.tsx

```
@@ -0,0 +1,536 @@
+      } else if (this.buffer[i] === '}') {
+        curlyBraceCount--;
+        if (curlyBraceCount === 0 && startIndex !== -1) {
+          try {
+            const jsonStr = this.buffer.substring(startIndex, i + 1);
+            const parsed = JSON.parse(jsonStr) as StreamResponseChunk;
+            return {
+              parsed,
+              remaining: this.buffer.substring(i + 1),
+            };
+          } catch (e) {
+            continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)